framework/widgets/Pjax.php

```diff
@@ -99,13 +99,13 @@ class Pjax extends Widget
     public $clientOptions;
     /**
      * @inheritdoc
-     * Note: This variable is redeclared to force separate counting for Pjax widgets (issue #12969)
-     * @since 2.0.11
+     * @internal
+     * @since 2.0.11 (#12969)
```

Comment:
@since: doesnt it matter that the counter is reset to 0 for Pjax Widgets?
@internal is copied from the Pjax.php to be consistent. IMO this variable should be private in the first place. Why should automatic counters be altered anyway?